packages/openapi-fetch/src/index.d.ts

```diff
@@ -15,7 +15,7 @@ export interface ClientOptions extends Omit<RequestInit, "headers"> {
   /** set the common root URL for all API requests */
   baseUrl?: string;
   /** custom fetch (defaults to globalThis.fetch) */
-  fetch?: (input: string, init?: RequestInit) => Promise<Response>;
+  fetch?: (input: Request) => Promise<Response>;
```

Comment:
This is _technically_ a breaking change, but since it was a breaking change from `0.9.0` I’m just going to pretend this was an accidental regression and never happened 🙈